src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -81,6 +84,8 @@ public WorkspaceDescription getWorkspace(UUID id) {
         desc.setSpendProfile(UUID.fromString(queryOutput.get("spend_profile").toString()));
       }
 
+      desc.setStage(WorkspaceStageEnum.fromValue(queryOutput.get("workspace_stage").toString()));
```

Comment:
similarly, not yet but I still strongly feel that we need to change this DAO to return something other than the `get workspace` request response. e.g., we're going to want to check the stage of a workspace to disable APIs. Why does that internal action generate a `get workspace` request response.
Doesn't have to be this PR, but it's fast approaching.

Reply:
I actually think this endpoint in particular is less of an issue, other than the continued thread of sharing API models with DB models. It's just pulling out every column of a workspace in the table, which we re-use directly as the response to a `getWorkspace` request.
It might make sense to create a specific method for just getting the enum if that's a concern, though